src/Makefile

```diff
@@ -128,7 +128,7 @@ tags:
 	ctags -R
 
 clean:
-	rm -f $(objects) $(deps)
+	rm -f $(objects) $(deps) .version.cc .version.o
```

Comment:
I am not sure we want to remove .version.cc as this is distributed in releases and wont get regenerated out of a git repository, I think we only want to remove .version.o

Reply:
> I am not sure we want to remove .version.cc as this is distributed in releases

No, `.version` is added to the release tarball, but `.version.cc` isn't. It can always be generated from `.version` (if available) or `git decribe`.
My main motivation is to make `bear -- make clean all debug=yes` behave as expected, so I don't care so much about `.version.cc`, we can also keep it

Reply:
Oh, right, my bad I misremembered how we managed version.cc then this commit is correct, no need to change anything.